WalletWasabi.Fluent/Global.cs

```diff
@@ -135,6 +137,8 @@ public async Task InitializeNoWalletAsync(TerminateService terminateService)
 
 			try
 			{
+				await StartupHelper.ModifyStartupSettingAsync(UiConfig.RunOnSystemStartup).ConfigureAwait(false);
```

Comment:
Just in case the user is actually changing these values outside Wasabi settings, to always doublecheck if the value is correct.
I can simplify this to run the first time only with `if (UiConfig.Oobe)`.

Reply:
Gotit. Do you think it's worth simplifying to first-time-run only?

Reply:
This was put here to make sure when you run the software for the first time, register the app for auto-start. This is the desired behavior. Everything else is just consequences. 

Sorry but the comment here is not correct in this sense.